registry/registry_internals.c

```diff
@@ -126,6 +126,11 @@ REGISTRY_PERSON_URL *registry_verify_request(char *person_guid, char *machine_gu
         info("Registry Request Verification: URL not found for person, person: '%s', machine '%s', url '%s'", person_guid, machine_guid, url);
         return NULL;
     }
+    //else if (pu->machine != m) {
```

Comment:
What is this commented block?

Reply:
We were considering adding it with @ktsaou, but decided to leave it commented out for now. The registry code will need a major revisit at some point, it's just that now is not the time.